tests/unit/aiplatform/test_training_jobs.py

```diff
@@ -31,6 +62,33 @@
 """
 _TEST_REQUIREMENTS = ["pandas", "numpy", "tensorflow"]
 
+_TEST_DISPLAY_NAME = "test-display-name"
+_TEST_TRAINING_CONTAINER_IMAGE = "gcr.io/test-training/container:image"
+_TEST_SERVING_CONTAINER_IMAGE = "gcr.io/test-serving/container:image"
+_TEST_SERVING_CONTAINER_PREDICTION_ROUTE = "predict"
+_TEST_SERVING_CONTAINER_HEALTH_ROUTE = "metadata"
+
+_TEST_DATASET_NAME = "test-dataset-name"
+_TEST_BASE_OUTPUT_DIR = "gs://test-base-output-dir"
+_TEST_RUN_ARGS = ["-v", 0.1, "--test=arg"]
+_TEST_REPLICA_COUNT = 1
+_TEST_MACHINE_TYPE = "n1-standard-4"
+_TEST_ACCELERATOR_TYPE = "NVIDIA_TESLA_K80"
+_TEST_INVALID_ACCELERATOR_TYPE = "NVIDIA_DOES_NOT_EXIST"
+_TEST_ACCELERATOR_COUNT = 1
+_TEST_MODEL_DISPLAY_NAME = "model-display-name"
+_TEST_TRAINING_FRACTION_SPLIT = 0.6
+_TEST_VALIDATION_FRACTION_SPLIT = 0.2
+_TEST_TEST_FRACTION_SPLIT = 0.2
+
+_TEST_OUTPUT_PYTHON_PACKAGE_PATH = "gs://test/ouput/python/trainer.tar.gz"
+
+_TEST_MODEL_NAME = "projects/my-project/locations/us-central1/models/12345"
+
+_TEST_PIPELINE_RESOURCE_NAME = (
```

Comment:
for later: all of these are getting a little heavy and difficult to manage.  we should at least put all of these pieces in a separate file so they are shared across tests.  further down the road we might need to modify the approach.